src/components/ecosystem/plugins.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+const plugins = require('./plugins.json')
+
+
+class PluginsTableRow extends React.Component {
+    constructor(props) {
+        super(props)
+        this.plugin = props.plugin
+    }
+
+    render() {
+        const nameCondition = this.props.nameFilter == undefined || this.plugin.name.includes(this.props.nameFilter)
+        const descriptionCondition = this.props.descriptionFilter == undefined || this.plugin.description.includes(this.props.descriptionFilter)
+
+        if (nameCondition && descriptionCondition) {
+            return (<tr>
+                <td><a href={this.plugin.url}>{this.plugin.name}</a></td>
+                <td>{this.plugin.description}</td>
+            </tr>)
+        }
+        else return null
+
+    }
+}
+
+export class PluginsTable extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            nameFilter: undefined,
+            descriptionFilter: undefined
+        }
+
+        this.filterName = this.filterName.bind(this);
+        this.filterDescription = this.filterDescription.bind(this);
+    }
+
+    filterName(event) {
+        this.setState({ nameFilter: event.target.value })
+    }
+
+    filterDescription(event) {
+        this.setState({ descriptionFilter: event.target.value })
+    }
+
+    render() {
+        return (
+            <table>
```

Comment:
I skipped using react table also because i neither wanted to add plugins for just a simple table.
Going to table a look at the CSS grids and implement that

Reply:
Done